src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -347,6 +366,7 @@ export default compose(
     withLocalize,
     withWindowDimensions,
     withFullPolicy,
+    withNetwork(),
```

Comment:
Same comment, can you add the prop types for `network` at the top of the file please?